### PR TITLE
Add Java 21 to universal container

### PR DESCRIPTION
**Pull request by timward60**
Adds Java 21 as a feature to the universal container.

#770

### CODE REVIEW

File: src/universal/.devcontainer/devcontainer.json [hunk 1]
```diff
@@ -43,8 +43,8 @@
             "additionalVersions": "3.1"
         },
         "ghcr.io/devcontainers/features/java:1": {
-            "version": "17",
-            "additionalVersions": "11",
+            "version": "21",
+            "additionalVersions": "11,17",
```
Comment (timward60) [inline src/universal/.devcontainer/devcontainer.json:47]: **todo**: I cannot find an example of the expected format for a list here, looking for where this is defined.

Comment (samruddhikhandale) [inline src/universal/.devcontainer/devcontainer.json:47]:
```suggestion
            "additionalVersions": "17",
```

For the universal image, we only support two latest LTS versions, so 21 and 17 makes sense. Can we make this change? Thanks!

Comment (samruddhikhandale) [reply src/universal/.devcontainer/devcontainer.json:47]: `additionalVersions` is an experimental Feature option, but looks like this works.

**Review by samruddhikhandale — COMMENTED** [review]
Thanks for contributing this PR, left one minor comment.

**Comment by timward60** [comment]
@samruddhikhandale Thank you for the review. Applied your suggestion.

**Review by samruddhikhandale — APPROVED** [review]
Thank you! 🎉 

The test failure https://github.com/devcontainers/images/actions/runs/7065223046/job/19235282277?pr=865#step:3:41740 is similar to https://github.com/devcontainers/images/pull/864#issuecomment-1834322144.

**Pull request merged**